ecs-agent/daemonimages/csidriver/driver/node.go

```diff
@@ -154,19 +163,26 @@ func (d *nodeService) NodeStageVolume(ctx context.Context, req *csi.NodeStageVol
 		if part != "0" {
 			partition = part
 		} else {
-			klog.InfoS("NodeStageVolume: invalid partition config, will ignore.", "partition", part)
+			klog.V(4).InfoS("NodeStageVolume: invalid partition config, will ignore.", "partition", part)
 		}
 	}
 
+	klog.V(4).Infof("NodeStageVolume: configs initialized for task %s and volume %s, time taken: %f seconds", taskId, volumeID, time.Since(nodeStageStartTime).Seconds())
+
+	findevicepathTime := time.Now()
 	source, err := d.findDevicePath(devicePath, volumeID, partition)
 	if err != nil {
+		klog.V(4).Infof(" NodeStageVolume: findDevicePath failed for task %s and volume %s, time taken: %f seconds", taskId, volumeID, time.Since(findevicepathTime).Seconds())
 		return nil, status.Errorf(codes.Internal, "Failed to find device path %s. %v", devicePath, err)
 	}
+	klog.V(4).Infof(" NodeStageVolume: findDevicePath succeeded for task %s and volume %s, time taken: %f seconds", taskId, volumeID, time.Since(findevicepathTime).Seconds())
 
+	pathexiststime := time.Now()
 	exists, err := d.mounter.PathExists(target)
-	klog.InfoS("NodeStageVolume: path exists:", "exists", exists)
+	klog.V(4).Infof("NodeStageVolume: PathExists for task %s and volume %s, time taken: %f seconds", taskId, volumeID, time.Since(pathexiststime).Seconds())
+	klog.V(4).InfoS("NodeStageVolume: path exists:", "exists", exists)
 	if err != nil {
-		klog.InfoS("NodeStageVolume: path exists:", "err", err)
+		klog.V(4).InfoS("NodeStageVolume: path exists:", "err", err)
```

Comment:
should this "path doesn't exist", if there's a err?

Reply:
Fixed in [502174e](https://github.com/aws/amazon-ecs-agent/pull/4326/commits/502174ee6c28610cf856718ec299d900f98eb65b)